frontend/package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "frontend",
+  "name": "vintage_turntable",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
   "dependencies": {
-    "@react-three/drei": "^8.16.7",
-    "@react-three/fiber": "^7.0.26",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
+    "@react-spring/three": "^9.5.5",
+    "@react-three/drei": "9.14.3",
+    "@react-three/fiber": "^8.9.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^0.26.0",
-    "bootstrap": "^5.1.3",
-    "dotenv": "^16.0.0",
-    "react": "^17.0.2",
-    "react-bootstrap": "^2.1.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "5.0.0",
-    "react-spotify-web-playback": "^0.9.1",
-    "spotify-web-api-node": "^5.0.2",
-    "three": "^0.138.3",
+    "framer-motion-3d": "^7.6.2",
```

Comment:
What is this used for?

Reply:
[Framer motion](https://www.framer.com/motion/) is another React animating library. I was thinking of using it for the GUI portion (ie. Spotify / Apple Music login button) of the front-end.